src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -190,38 +194,45 @@ void parse_processor_info_win(const char* base_ptr,
                     _cpu_mapping_table[list[m] + base_proc][CPU_MAP_GROUP_ID] = group_id;
                     _proc_type_table[0][EFFICIENT_CORE_PROC]++;
                 }
-            } else if ((2 == list_len) && (-1 == _cpu_mapping_table[list[0] + base_proc][CPU_MAP_CORE_TYPE])) {
+            } else if (2 == list_len) {
+                if (-1 == _cpu_mapping_table[list[0] + base_proc][CPU_MAP_CORE_TYPE]) {
```

Comment:
Could be `constexpr` variables created for magic values `2`, `-1`? Will better explain the condition.

Reply:
@praasz Updated. Please review again.